tests/sentry/api/endpoints/test_team_projects.py

```diff
@@ -168,18 +167,16 @@ def test_disable_member_project_creation(self):
         self.create_member(
             user=test_member,
             organization=test_org,
-            role="admin",
-            teams=[test_team],
+            role="member",
             team_roles=[(test_team, "contributor")],
         )
         self.login_as(user=test_member)
-        response = self.get_error_response(
+        self.get_error_response(
             test_org.slug,
             test_team.slug,
             **self.data,
             status_code=403,
         )
-        assert response.data["detail"] == DISABLED_FEATURE_ERROR_STRING
```

Comment:
why aren't we checking for this anymore?

Reply:
I wrote this test wrong originally, when the org member is has the `member` role they actually get a permission denied error since they don't have the required scopes to access this endpoint at all